litellm/integrations/traceloop.py

```diff
@@ -1,10 +1,26 @@
+import traceback
+
+
 class TraceloopLogger:
     def __init__(self):
-        from traceloop.sdk.tracing.tracing import TracerWrapper
-        from traceloop.sdk import Traceloop
+        try:
+            from traceloop.sdk.tracing.tracing import TracerWrapper
+            from traceloop.sdk import Traceloop
+
+            Traceloop.init(app_name="Litellm-Server", disable_batch=True)
```

Comment:
If we are tryin to catch and present a different message in the console if there is no traceloop package, adding any extra statement inside the try:except might confuse user when exception occurs from those extra statements.
```python
Traceloop.init(app_name="Litellm-Server", disable_batch=True)
self.tracer_wrapper = TracerWrapper()
```